scaleapi/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.2"
```

Comment:
Oh not sure my comment here got saved, need to bump to `1.0.3`

Reply:
```suggestion
__version__ = "1.0.3"
```